decidim-core/spec/forms/notifications_settings_form_spec.rb

```diff
@@ -178,7 +178,8 @@ module Decidim
     describe "#meet_push_notifications_requirements?" do
       context "when the notifications requirements are met" do
         before do
-          Rails.application.secrets[:vapid] = { enabled: true }
+          allow(ENV).to receive(:fetch).and_call_original
+          allow(ENV).to receive(:fetch).with("VAPID_PUBLIC_KEY", nil).and_return("FOO BAR")
```

Comment:
Let's change it to a stub for `Decidim.vapid_public_key`

Reply:
Fixed in 22115a5511